.github/workflows/continuous-build.yml

```diff
@@ -0,0 +1,62 @@
+name: Continuous Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build on ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
```

Comment:
I know that choosing non-AMD platforms isn't supported with GH hosted runners yet, but is it worth/possible to copy your trick with QEMU below to also test on an (emulated) ARM platform here?

Reply:
The qemu setup there I think is special to docker, or at least I'd need to dive deep on what it's doing - the action is hosted by docker at least. Running tests on arm sounds like it'd be nice let's follow up on how we can do it.